doc/sphinx.txt

```diff
@@ -4108,10 +4110,10 @@ properly.
 
 Second, we additionally need to know the language-specific string sorting
 rules. For instance, when sorting according to US rules in en_US locale,
-the accented character '�' (small letter i with diaeresis) should be placed
+the accented character '�' (small letter i with diaeresis) should be placed
```

Comment:
Konstantin,

I will look at what went wrong with the encoding on my side. In the meantime, can you please tell me what encoding those documentation files use ?

Thx.